### DISCUSSION TITLE
Better way to save EXIF data from TIFF in JPEG

### DISCUSSION THREAD

**Discussion by gverm** [python-pillow/Pillow#6927] [Q&A] [answered]
I have TIFF images that I read the EXIF data from and then convert those to JPEG and I'd like a way to preserve as much of the original EXIF data as possible. Currently I do it like this:

```py
src_img = Image.open(input_file)
exif = src_img.getexif()
for key, value in exif.items():
    if isinstance(value, tuple):
        del exif[key]
dst_img = ImageOps.exif_transpose(src_img).convert("RGB")
output_file = io.BytesIO()
dst_img.save(output_file, "jpeg", quality=95, exif=exif)
```

I'm removing the tuple values because otherwise I get an Exception `NotImplementedError: multistrip support not yet implemented`.

Is there a better way to do this? Or should I just not do this at all?

**Comment by radarhere** [comment, marked as answer]
Rather than deleting all of the tuple values, you should be able to just delete the STRIPOFFSETS tag.

```python
from PIL import TiffImagePlugin
del exif[TiffImagePlugin.STRIPOFFSETS]
```

**Reply by gverm** [reply]
Thanks, that works perfectly!
